tests/FunctionPathTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @copyright 2018 Hostnet B.V.
+ */
+declare(strict_types=1);
+
+class FunctionPathTest extends \PHPUnit\Framework\TestCase
+{
+    public function testRun()
+    {
+        $current_location = getcwd();
+        $expected_results = [
+            $current_location."/test_files/A.php::test1",
+            $current_location."/test_files/A.php::inTest1",
+            $current_location."/test_files/A.php::test2",
+            $current_location."/test_files/ClassB.php/ClassB::__construct",
+            $current_location."/test_files/ClassB.php/ClassB::test3",
+            $current_location."/test_files/ClassB.php/ClassB::inTest3",
+            $current_location."/test_files/ClassC.php/Dead\TestNamespace/ClassC::test4",
+            $current_location."/test_files/ClassC.php/Dead\TestNamespace::test5",
+            $current_location."/test_files/ClassC.php/Dead\TestNamespace/ClassD::test6",
```

Comment:
This seems like an odd result. I would expect only to use `/` be used as filename separator.
I would expect something like `"/test_files/ClassC.php::Dead\TestNamespace\ClassD::test6"`. Becaue you have `/test_files/ClassC.php` as file part `Dead\TestNamespace\ClassD` as FQNS and `::test6` as method/function.